acceptance/assertions/image.go

```diff
@@ -1,3 +1,4 @@
+//go:build acceptance
 // +build acceptance
```

Comment:
I see from this that we need to have both the `go:build` and `+build` lines: https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

Reply:
Correct. It's worth noting that `go fmt` on 1.17 does this for us automatically.

See https://golang.org/doc/go1.17#go-command